### ISSUE TITLE
iZettle payment error: Wrong currency.

### ISSUE THREAD

**Issue by 34xi** [iZettle/sdk-ios#136]
Hi all, I have en issue with the latest version of iZettle. I had similar issue about 2 months ago after update to 1.4.0 so probably it's similar.

After login and trying to pay iZettle shows error screen with `Wrong currency` message (and sad face). I checked the code and found that I sent `EUR` currency code.

Last time it happened my college talked with your support and problem disappeared.

Thanks in advance.

Environment:
iZettle SDK for iOS v 1.4.2
iOS v10.0.2
Demo account

**Comment by mattiasjahnke** [comment]
Hi @34xi 
Are you sure that the account is using EUR? If you don't specify `EUR` as currency code, you should be able to see what currency the account is configured for. Please double check that it's EUR.

**Comment by 34xi** [comment]
Hi @mattiasjahnke,
thank you for your reply.

No, I am not sure that `account` uses `EUR` currency, but I am sure that it worked before update like at previous time (worked before update, then shows this error after update).

And I am sure that app (and SDK) used `EUR` as a parameter for 
`- (void)chargeAmount:(NSDecimalNumber *)amount
            currency:(nullable NSString *)currency
           reference:(nullable NSString *)reference
presentFromViewController:(UIViewController *)viewController
          completion:(iZettleSDKOperationCompletion)completion;
` as currency few times. I checked it many times, not only twice. Then you blocked our demo account, then gave us new credentials and now it works.

It looks like previous time - from our side was changed nothing and it just started to work, so thank you for the fix.
